hc-tree-view-behavior.html

```diff
@@ -0,0 +1,141 @@
+<link rel="import" href="../polymer/polymer.html">
+
+<script>
+  /* global hc */
+  // Ensure our namespace exists
+  window.hc = window.hc || {};
+
+  /**
+   * Use `hc.NestedListBehavior` to implement a flat data structure of nested items into a list with
+   *  children getting added/remove to emulate open/close functionality.
+   *
+   * @polymerBehavior hc.NestedListBehavior
+   */
+  hc.TreeViewBehavior = {
+    properties: {
+      itemIdName: {
+        type: String,
+        value: 'id'
+      },
+      itemParentIdName: {
+        type: String,
+        value: 'parentId'
+      },
+      /* The list of all selected items without duplicates */
+      selectedItems: {
+        type: Array,
+        value: () => []
+      },
+      /* The parts object used by the list element */
+      _parts: Object
```

Comment:
What's the benefit of this annotation?  By default they are marked as private in their docs by using the leading `_`.